src/assets/styles/10-settings/_typography.scss

```diff
@@ -41,8 +41,9 @@
   --heading-line-height: var(--body-line-height);
 
   // letter spacing
-  --body-letter-spacing: 0.03125rem;
-  --heading-letter-spacing: -0.014em;
+  --body-letter-spacing: 0.4px;
+  --heading-bold-letter-spacing: -0.5px;
```

Comment:
What is your reasoning behind using "bold" in the name?

Reply:
We are using this variable only for headings with Wellcome Bold font. Wanted to make it clear that we are using it for this reason...

Reply:
Happy to change it!

Reply:
The word "bold" is tied in too much with heading style and especially as they are now (using Wellcome-Bold). I would've suggested `--heading-primary-letter-spacing` but when it comes to font families Wellcome-Bold is actually set as `--font-secondary` so that seems odd.

@chriddell any thoughts?

Reply:
Hmm, I agree that using "bold" ties us to a "bold" font (which may change) and is also confusing as our font is literally named "Bold", too....

I can see we're using 3 letter-spacing values, so how about we tie their names to the scenarios they are used in? Taking the "Wellcome Bold" font as an example - what is the scenario(s) in which we use "Wellcome Bold"? e.g. is it all h1s and h2s? Is it when they are used in certain components? Is it when a heading is above a certain font-size?

Presumably there is a rule for when to use Wellcome Bold, and when to use Helvetica. If we know the rule, we can name our letter-spacing accordingly.

Would ideally end up something like

```
--body-letter-spacing: xx;
--heading-large-letter-spacing: xx; // equivalent to --heading-bold-letter-spacing
--heading-small-letter-spacing: xx;
```

Reply:
My understanding is that only that theoretical <h0> and <h1> are using Wellcome Bold and only these two headings have letter-spacing **-0.5px**.